static/main.js

```diff
@@ -24,7 +24,7 @@
 "use strict";
 
 require("monaco-loader")().then(function () {
-
+    require('popper.js');
```

Comment:
What does the "popper" lib do? I don't see how it's being used here.

Reply:
It's a bootstrap 4 dependency, to handle popup positionings. I think it loads it internally, so this line might be redundant, yes